src/darwin/Framework/CHIP/CHIPDevice.mm

```diff
@@ -82,15 +91,15 @@ - (BOOL)sendMessage:(NSData *)message error:(NSError * __autoreleasing *)error
     return YES;
 }
 
-- (BOOL)sendCHIPCommand:(uint32_t (^)(chip::System::PacketBuffer *, uint16_t))encodeCommandBlock
+- (BOOL)sendCHIPCommand:(uint32_t (^)(chip::System::PacketBufferHandle &, uint16_t))encodeCommandBlock
```

Comment:
That first arg should be a `const chip::System::PacketBufferHandle &` is this is a borrow, not an ownership transfer.